temp-scripts/fix-all-linting.sh

```
@@ -0,0 +1,92 @@
+  wait $pid
+done
+
+echo ""
+echo "================================================"
+echo "✅ Linting fixes attempted for all packages"
+echo ""
+echo "Now running final lint check..."
+echo ""
+
+# Final check
+cd /Users/MAC/Documents/projects/caia
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)